hr_expense_sequence/models/hr_expense_sheet.py

```diff
@@ -9,11 +9,11 @@ class HrExpenseSheet(models.Model):
     _inherit = "hr.expense.sheet"
     _rec_name = "number"
 
-    number = fields.Char(required=True, default="/", readonly=True, copy=False)
+    number = fields.Char(default=None, readonly=True, copy=False)
```

Comment:
why are you delete required and change default value?

Reply:
**-delete required**
Because tests / test with Odoo (pull_request) will occurred
`psycopg2.errors.NotNullViolation: null value in column "number" violates not-null constraint`
If you set number fields required=True 

**-change default value**
At first PR I try to fix 
`psycopg2.errors.NotNullViolation: null value in column "number" violates not-null constraint`
By change default value to **None** but it not worked and I didnot change it back.
So I decided to delete required and it passed.

After all this is not effect to the module's functional.
or should I fix anything?